resources/js/Pages/Results.vue

```diff
@@ -420,15 +421,15 @@ h2 {
 	}
 }
 .mismatches-form-footer {
-    margin-top: $dimension-layout-xsmall;
+    margin-top: var(--dimension-layout-xsmall);
     display: flex;
     flex-direction: row-reverse;
     justify-content: space-between;
     align-items: flex-start;
-    gap: $dimension-layout-xsmall;
+    gap: var(--dimension-layout-xsmall);
     // calculate the footer height to reserve space for
     // messages with two lines (1.5 line height plus padding)
-    min-height: calc(2*1.5em + 2*$dimension-spacing-large);
+    min-height: calc(2*1.5em + 2*var(--dimension-spacing-large));
```

Comment:
```suggestion
    min-height: calc(2*var(--dimension-spacing-xlarge) + 2*var(--dimension-spacing-large));
```

Reply:
only if it makes sense?

Reply:
for the sake of avoiding git complications i added the changes manually instead of committing your changes. Those should be correct now.